### ISSUE TITLE
tsconfig.json excludes packages

### ISSUE THREAD

**Issue by hiporox** [hero-org/herocast#214]
Currently the `tsconfig.json` excludes all of `node_modules`, making type hinting for all of the installed packages except the listed ones (`react/next`, `react-dom/next`, `node`) not work. This makes development very hard, as most of the advantages of using typescript language server type hinting don't work. An easy fix is to remove the `exclude` section from the config file.

I am opening this as an issue instead of a PR to promote a better discussion around why this choice was made at first

**Comment by hellno** [comment]
<img width="1167" alt="image" src="https://github.com/hellno/herocast/assets/686075/4f5ae265-3c9f-41cb-8fe4-074481d0a436">

@hiporox I tried fixing it, but default running adds back the option:
```
"exclude": [
    "node_modules"
  ]
  ```

**Comment by scottrepreneur** [comment]
Is this revealing itself to you somehow, @hiporox ? I'm seeing the same as hellno here.

I was getting a crash on the ts server with the types listed. Changing to these helped for me locally: 

```      ~~"react/next",~~
      ~~"react-dom/next"~~
      "react",
      "react-dom",
      "next"```

**Comment by hiporox** [comment]
@hellno I am not sure why, but that error just started appearing for me too, when it was not when I made this PR. At the time I made this, VSCode wouldn't import any of the type declarations from any of the `node_modules` files. But now it is importing them fine with the exclude added, and Next.js is showing the same error that you are showing

@scottrepreneur The change you suggested does solve a problem of VSCode not being able to locate the `react/next` and `react-dom/next` imports

**Comment by hellno** [comment]
cool! thanks for clarifying this. I'm not in the details of the `tsconfig`. 
Would you be up to create a PR which reverts my change and adds your fix, @scottrepreneur ? 😊

**Comment by scottrepreneur** [comment]
Should be closed by #237
